lib/ljglibs/gio/file.moon

```diff
@@ -57,6 +66,17 @@ core.define 'GFile', {
   enumerate_children: (attributes, flags = @QUERY_INFO_NONE) =>
     gc_ptr catch_error C.g_file_enumerate_children, @, attributes, flags, nil
 
+  copy: (dest, flags, cancellable, callback) =>
+    self = @
+    local cb_handle
+
+    handler = (current_bytes, total_bytes) ->
+      callback self, current_bytes, total_bytes
+
+    cb_handle = callbacks.register handler, 'file-progress'
```

Comment:
There's no check for whether "handler" actually is present. In this PR, it actually isn't so there's an ignored error raised for every file save.